errors/react-hydration-error.md

```diff
@@ -82,6 +82,10 @@ Common causes with css-in-js libraries:
 - When using other css-in-js libraries
   - Similar to Styled Components / Emotion css-in-js libraries generally need configuration specified in their examples in the [examples directory](https://github.com/vercel/next.js/tree/canary/examples)
 
+#### Local Overrides
+
+It's possible you may have Local Overrides enabled in Chrome devtools. With this enabled, the HTML served will be different from what the SSR emitted. It also won't show up in view-source, so you may be left wondering what is going on
```

Comment:
Is it possible to describe this section in more detail?

E.g., Will Local Overrides always cause Hydration Mismatch? What kind of files will cause the hydration mismatch after being edited and persisted in DevTools? What should the developer do to clean up the Local Overrides?